test/secret/secret_test.go

```diff
@@ -120,6 +120,58 @@ var _ = Describe("secret test", func() {
 
 			// Verify that Pass4SymmKey is updated
 			testenv.VerifySecretsUpdatedOnPod(deployment, testenvInstance, verificationPods, testenv.SecretObject["Pass4SymmKey"], testenv.DecodeBase64(secretObj.Data.Pass4SymmKey))
+
+			// Delete secret key
+			testenv.UpdateSecret(deployment, testenvInstance.GetName(), secretObj, true)
```

Comment:
Add a check if the operation was successful.
```
err := testenv.UpdateSecret(deployment, testenvInstance.GetName(), secretObj, true /*delete*/)
Expect(err).To(Succeed(), "Unable to delete secret Object")
``` 

This will fail the test if update is not successful.

Reply:
updated